docker-app/qfieldcloud/core/models.py

```diff
@@ -1284,8 +1284,9 @@ class Meta:
     # projects with large or numerous attachments.
     is_attachment_download_on_demand = models.BooleanField(
         default=False,
+        verbose_name="On demand attachment files download",
```

Comment:
```suggestion
        verbose_name=_("On demand attachment files download"),
```

Translation missing.

Reply:
@suricactus , _( == tr( ? TIL again :)